public/js/applicantModal.js

```diff
@@ -40,11 +40,19 @@ function addCurrency(number) {
     }
     return 'Unknown'
 }
-
+function copyToClipboard(element) {
+    var $temp = $("<input>");
+    $("body").append($temp);
+    $temp.val($(element).text()).select();
+    document.execCommand("copy");
+    $temp.remove();
+    document.querySelector("button.clipboard").innerText = 'Copied';
+}
 
 $(document).ready(function(){
     $(".myBtn").click(function(){
         var url = './api/getApplicant/' + this.dataset.id
+        let studentUrl = 'http://localhost:8080/public/'+this.dataset.id
```

Comment:
Switch to a var for consistency sake - even though let is better

Reply:
disagree, swap them both to lets, better than they found it over consistency